lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -89,29 +89,8 @@ private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
 
-    $drupal_user = user_load($data->uid);
-
-    if (!isset($data->field_northstar_id_value) || $data->field_northstar_id_value === 'NONE') {
-      $northstar_response = dosomething_northstar_get_northstar_user($data->uid);
-      $northstar_response = json_decode($northstar_response);
-
-      if (!empty($northstar_response->data) && !isset($northstar_response->error)) {
-        $northstar_user = $northstar_response->data;
-        user_save($drupal_user, ['field_northstar_id' => [LANGUAGE_NONE => [0 => ['value' => $northstar_user->id]]]]);
-      }
-    }
-
-    $last_initial = substr(dosomething_helpers_extract_field_data($drupal_user->field_last_name), 0, 1);
-    if ($last_initial === FALSE) {
-      $last_initial = NULL;
-    }
-
     $this->user = [
```

Comment:
Ok, so just to be sure on this (since Gladiator highly depends on this part at the moment), the response sends back to Northstar the drupal id, and Northstar figures out the rest to add the user info, etc?

Gonna cc @deadlybutter cause he worked on the above fix for Gladiator. Once this is merged we should definitely test out Gladiator QA w/ Phoenix Staging!